### ISSUE TITLE
Possible bug

### ISSUE THREAD

**Issue by idiazst** [nt-williams/lcm#1]
In this line we reassign D_M to be equal to I(M=-m) * D_M. But computation of the variance requires D_M, not I(M=m) * D_M. Can this be causing problems?

https://github.com/nt-williams/lcm/blob/a04eb60660420b2c7a7fa735c3fff89c7153a726/lcm/R/CrossFit_D_Zt_Mt.R#L19

**Comment by nt-williams** [comment]
The re-assignment is only occurring in the training set `Tr_a`, not `P_a` which is what is ultimately used for estimating the variance. Maybe I'm missing something though?

**Comment by idiazst** [comment]
Yes, I think you are right. But I do think it is cleaner to create another object for the outcome of this regression and keep D_M with its original definition. Might save us other headaches.
